x-pack/plugins/actions/server/builtin_action_types/email.test.ts

```diff
@@ -454,21 +482,9 @@ describe('execute()', () => {
         "status": "ok",
       }
     `);
+    delete sendEmailMock.mock.calls[0][1].configurationUtilities;
```

Comment:
This is where I delete the configuration utilities from the calls structure, so we no longer compare it, so we don't have to change this code when we add new methods to the configuration utilities.